db/fixtures/server_roles.csv

```diff
@@ -2,6 +2,7 @@ name,description,max_concurrent,external_failover,role_scope
 automate,Automation Engine,0,false,region
 database_operations,Database Operations,0,false,region
 database_owner,Database Owner,1,false,database
+embedded_ansible,Embedded Ansible,1,false,zone
```

Comment:
Yeah, the only downside I can see to using an explicit `region` role is that it's different than all of the other provider type roles.  So, while we're trying to treat it as a provider, it's not quite one.  I can't think of any other problems with that though, migrating to zonal role should be straight forward.